system-tests/playwright.config.ts

```diff
@@ -11,7 +11,9 @@ function envToNumber(env: string, defaultNumber: number) {
 const config: PlaywrightTestConfig = {
   globalSetup: require.resolve("./src/setup/globalSetup.ts"),
   globalTeardown: require.resolve("./src/setup/globalTeardown.ts"),
-  use: {},
+  use: {
+    headless: true,
```

Comment:
make sure this is false when using the --headed flag

Reply:
it works because headed flag will override this